Dockerfile

```diff
@@ -11,43 +11,37 @@ USER 0
 COPY package.json yarn.lock lerna.json tsconfig.json .prettierrc ./
```

Comment:
why do we have to copy all these separately if they're not being renamed and put in different directories?  if there are things we don't want to have copied over (such as a lot of config stuff in the top level of the repo), it seems like we should expand the .dockerignore accordingly.  and even then, why is it necessary to have some of these config files like the prettier one?  would the build fail due to some sorta linting issue if it wasn't present?

Reply:
I'll agree prettier probably shouldn't need to be there - the rest seem to be updated regularly. Package.json, yarn.lock, and tsconfig.json are dependencies or config files for dependencies, which would probably/definitely break things if removed. Lerna seems to be for the monorepo, which I'm not experienced with, and .prettierc can probably be removed and added to the .gitignore without issue.

As far as copying them all separately from everything else, they could theoretically be in their own folder named 'dependencies' but then they'd probably simply be copied to the root of the container anyways, so I don't see much value in doing that for the sake of Dockerfile performance. Repo housekeeping is a different story.